UniVRM 0.129.0 is no longer compatible with Unity 2021 and 2022.1

Commit 3d8db45 broke compatibility with older Unity versions because the `NativeArray::AsSpan` method is only [recently implemented in 2022.2](https://docs.unity3d.com/2022.2/Documentation/ScriptReference/Unity.Collections.NativeArray_1.html) but [not available in 2021.1 and older versions](https://docs.unity3d.com/2022.1/Documentation/ScriptReference/Unity.Collections.NativeArray_1.html)

`Library\PackageCache\com.vrmc.gltf@027e57fe26\Runtime\UniGLTF\IO\NativeArrayManager.cs(72,26): error CS1929: 'NativeArray<T>' does not contain a definition for 'AsSpan' and the best extension method overload 'MemoryExtensions.AsSpan(string)' requires a receiver of type 'string'`

--- a/Assets/UniGLTF/Runtime/UniGLTF/IO/NativeArrayManager.cs
+++ b/Assets/UniGLTF/Runtime/UniGLTF/IO/NativeArrayManager.cs
@@ -69,9 +69,14 @@ namespace UniGLTF
         public NativeArray<T> CreateNativeArray<T>(ArraySegment<T> data) where T : struct
         {
             var array = CreateNativeArray<T>(data.Count);
+#if UNITY_2022_2_OR_NEWER
             var toSpan = array.AsSpan();
             var fromSpan = data.AsSpan();
             fromSpan.CopyTo(toSpan);
+#else
+            for (int i = 0; i < data.Count; i++)
+                array[i] = data.Array[data.Offset + i];
+#endif
             return array;
         }
 

